include/deal.II/particles/particle_handler.h

```diff
@@ -270,6 +287,20 @@ namespace Particles
       const Particle<dim, spacedim> &particle,
       const typename Triangulation<dim, spacedim>::active_cell_iterator &cell);
 
+    /**
+     * Insert a particle into the collection of particles given all the
+     * properties necessary for a particle. This function is used internally to
```

Comment:
Could you slightly expand on what the user should think of the word *internally* here, given that the function is now public? It would also be good if `particle_index` is explained more clearly in terms of the expectations from outside.